src/front/ticketer/lib/backend_communication/logic/communication.dart

```diff
@@ -119,6 +119,7 @@ class BackendCommunication {
         throw ArgumentError("Token is not valid");
       }
       dio.options.headers.addAll({"Authorization": "Bearer ${token.token}"});
+      dio.options.contentType = Headers.jsonContentType;
```

Comment:
Could you please add the same line in standard postCall method? Just as a workaround for login bugs, we will implement nicer solution in TM-84